Weather/Views/MenuHeaderView.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  MenuHeaderView.swift
+//  Weather
+//
+//  Created by Alex on 4.08.22.
+//
+
+import SwiftUI
+import Lottie
+
+struct MenuHeaderView: View {
+    
+    @ObservedObject var cityViewModel: CityViewModel
+    @State private var searchTerm = "Minsk"
+    
+    var body: some View {
+        HStack {
+            TextField("", text: $searchTerm)
+                .padding(.leading, 20)
+            
+            Button {
+                cityViewModel.city = searchTerm
+            } label: {
+                ZStack {
+                    RoundedRectangle(cornerRadius: 10)
+                        .fill(Color.blue)
+                    Image(systemName: "location.fill")
+                }
+            }
+            .frame(width: 50, height: 50)
+        }
+        .foregroundColor(.white)
+        .padding()
+        .background(ZStack (alignment: .leading) {
+            Image(systemName: "magnifyingglass")
+                .foregroundColor(.white)
+                .padding(.leading, 10)
```

Comment:
10 is a default value, you can remove it: `.padding(.leading)`

Reply:
Fixed.)